.github/workflows/tests.yml

```diff
@@ -298,7 +298,7 @@ jobs:
         cd ..
         kubectl -n cluster-sample-ns patch deploy harborcluster-sample-harbor-harbor-core -p '{"spec":{"template":{"spec":{"containers":[{"name":"core","env":[{"name":"GC_TIME_WINDOW_HOURS","value":"0"}]}]}}}}'
         sleep 5
-        kubectl -n cluster-sample-ns wait --for=condition=Ready pod --all --timeout 600s
+        kubectl -n cluster-sample-ns wait --for=condition=Ready -l job-type!=minio-init pod --all --timeout 600s
```

Comment:
Why no need to wait for the job is completed? I'm thinking if starting the harbor cluster deployment when the job is not completed may lead to some error?

Or the minio condition status has taken into account the job status into its status?

Reply:
yes, we just check the minIO and registry components, if all of them works well, This indicates that the minIO-init job has completed